index.html

```diff
@@ -0,0 +1,160 @@
+          <ul class="tool1">
+            <li>CANOPY</li>
+            <li><img src="image/Counter.png" alt="dot"></li>
+            <li class="fint">Back End Dev</li>
+            <li><img src="image/Counter.png" alt="dot"></li>
+            <li class="fint">2015</li>
+          </ul>        
+          <p class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</p>
+          <ul class="tool2">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+          <button type="button" class="button detail-style2">See Project</button>
+        </div>
+      </div>
+      <div class="card">
+        <img src="image/Snapshoot-Portfolio-(2).png" alt="A photo of a website page">
+        <div class="card-content">
+          <h2 class="card-heading">Tonic</h2>
+          <ul class="tool1">
+            <li>CANOPY</li>
+            <li><img src="image/Counter.png" alt="dot"></li>
+            <li class="fint">Back End Dev</li>
+            <li><img src="image/Counter.png" alt="dot"></li>
+            <li class="fint">2015</li>
+          </ul>        
+          <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
+          <ul class="tool2">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+          <button type="button" class="button detail-style2">See Project</button>
+        </div>
+      </div>
+      <div class="card">
+        <img src="image/Snapshoot-Portfolio-(3).png" alt="A photo of a website page">
+        <div class="card-content">
+          <h2 class="card-heading">Multi-Post<br>Stories</h2>
+          <ul class="tool1">
+            <li>CANOPY</li>
+            <li><img src="image/Counter.png" alt="dot"></li>
+            <li class="fint">Back End Dev</li>
+            <li><img src="image/Counter.png" alt="dot"></li>
+            <li class="fint">2015</li>
+          </ul>        
+          <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
+          <ul class="tool2">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+            <button type="button" class="button detail-style2">See Project</button>
+        </div>
+      </div>
+      </section>
+      <section class="about-section">
+        <div class="myself">
+          <h2>About<br>Myself</h2>
+          <p class="about-detail">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          <p><a href="" class="info-link">LET'S CONNECT</a></p>
```

Comment:
- [ ] Great job so far, although this should be a `hx (h2,h3...)` tag rather than a link inside in a paragraph tag, please fix that.